examples/container-service-extension-3.1.x/3.7-cse-3.1.x-install.tf

```diff
@@ -25,12 +26,142 @@ terraform {
   }
 }
 
+# We'll use the System Administrator only to create the service role, the service user to manage CSE and the minimum
```

Comment:
I think we should explain that this is provider alias (some users may miss it). In other words, provide some context on the approach.

Reply:
Done, let me know your thoughts.